### ISSUE TITLE
How to hidden marker after chart data change?

### ISSUE THREAD

**Issue by MarsLuo** [wuxudong/react-native-charts-wrapper#176]
<img width="344" alt="screen shot 2018-01-13 at 11 54 18 am" src="https://user-images.githubusercontent.com/1073431/34902565-20ac98be-f859-11e7-81a6-574b98ce8f25.png">

Hello, @wuxudong 。
I find that, when I load data and select a enety to make marker show, then I change the data, but the marker not hidden and I can' find a way to make it hidden.

Do you know about this?

**Comment by GermanMontejo** [comment]
if you don't need the marker anymore when you change the data, you can disable the marker via:
marker: {enable: false}

**Comment by darkfiredarkhalo** [comment]
The marker is necessary,but the position of the marker is not suitable...

**Comment by sairohithA007** [comment]
yeah... even I have the same requirement as @darkfiredarkhalo. I need the marker when I tap on a point in the line chart. When the data changes, the marker position is inappropriate. I tried using a boolean state variable for the marker, but that is not working as expected. In Android the rendering of the marker is fine, but on iPhone, I need to tap on the same point thrice to get the marker rendered.

**Comment by sairohithA007** [comment]
I don't know the reason, but my problem is resolved. Now, I'm able to observe that the marker is hidden whenever new data is rendered on the screen.

Thanks :)

**Comment by wuxudong** [comment]
@sairohithA007  How did you do that?

**Comment by wuxudong** [comment]
in 0.4.4, a new function is introduced. 

    this.refs.chart.highlights([])

this can be used to manually clear all the highlights.

![highlights](https://user-images.githubusercontent.com/1391101/43240796-cf109c34-90ca-11e8-88e2-e4e625cdaff8.gif)

**Comment by sairohithA007** [comment]
@wuxudong, as said earlier I tried with a state variable(boolean) as passed as value to marker prop. This did not work as expected. So, I removed this change and re-build the application.  Other than this I did not do anything.

**I got the below behavior:**
When the line chart is rendered and on tap on any point, the marker appeared. Next, the data for the line chart is changed and the line chart re-render and the marker disappeared from the view. Again when tapped on any point on line chart the marker is appearing.
